### PR TITLE
feat: Implemented Subtract function

### PR DESCRIPTION
**Pull request by ghost**
- Implemented Subtract Function
- Added JSDoc for Subtract function

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * This function calculates and returns the difference of two numbers
+* @param {number} num1 
+ * @param {number} num2 
+ * @returns result of num1 - num2
+ */
+function subtract(num1, num2){
```
Comment (FayazCodeCraft) [inline math.js]: Give descriptive variable names for better understanding

**Review by FayazCodeCraft — CHANGES_REQUESTED** [review]

File: math.js (hunk 1 above)
Comment (ghost) [reply math.js]: **Done**

**Review by FayazCodeCraft — APPROVED** [review]
LGTM

**Review by vinusuhas-codecraft — APPROVED** [review]
Nice but its squash time

**Pull request merged**